domains/_discord.lumi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "itsFatlum",
+        "email": ""
```

Comment:
I recommend adding your discord ID in the "discord" string, this should be how you should do it here:
```json
        "username": "itsFatlum",
        "email": "",
        "discord": "enterdiscordid"
```